src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIInteract.java

```diff
@@ -230,6 +231,15 @@ protected final boolean mineBlock(
         return true;
     }
 
+    /**
+     * Event triggered after receiving a list of block drops.
+     * @param blockDrops the received items from the block.
+     */
+    public void onBlockDropReception(final List<ItemStack> blockDrops)
+    {
+
```

Comment:
Empty method? Should this have a // No-op text?

Reply:
Default method, other override if needed.

Reply:
Still, maybe just a simple `// No-op` text in it like we do in the majority of places just as an indication this is purposefully left empty.